src/board.py

```diff
@@ -0,0 +1,68 @@
+import networkx as nx
+
+
+class Board:
```

Comment:
See with @JulienMattiussi about class, be prefer the functionnal approach

Reply:
Done, as far as there is a clean separation, it 's ok.

Reply:
Not a big deal for me to choose between functionnal or Classes in this case.
The important is to keep a logical structure and use it correctly
Just have in mind you won't be able to keep classes structure in laters sprints (ie in React) when it won't be relavant anymore

Reply:
Other advices on this point ?